Packs/MongoDB/Integrations/MongoDB/MongoDB.py

```diff
@@ -0,0 +1,347 @@
+from CommonServerPython import *
+
+from json import JSONDecodeError
+from typing import List, Union, Tuple
+
+from bson.objectid import ObjectId
+from pymongo import MongoClient
+from pymongo.collection import Collection
+from pymongo.database import Database
+from pymongo.errors import OperationFailure
+from pymongo.results import InsertManyResult, UpdateResult, DeleteResult
+
+CONTEXT_KEY = 'MongoDB.Entry(val._id === obj._id && obj.collection === val.collection)'
+
+
+class Client:
+    def __init__(
+        self,
+        urls: List[str],
+        username: str,
+        password: str,
+        database: str,
+        ssl: bool = False,
+        insecure: bool = False
+    ):
+        if insecure and not ssl:
+            return_error(f'"Trust any certificate (not secure)" must be ticked with "Use TLS/SSL secured connection"')
```

Comment:
why is this restriction? where is that documented on pymongo?

Reply:
when you're using them both, that's the error raised.

Reply:
what error?

Reply:
```
ConfigurationError: ssl has not been enabled but the following ssl parameters have been set: ssl_cert_reqs. Please set `ssl=True` or remove.
```